src/azul/terra.py

```diff
@@ -257,13 +264,17 @@ def _bigquery(self, project: str) -> bigquery.Client:
         with aws.service_account_credentials():
             return bigquery.Client(project=project)
 
+    def _trunc_query(self, query: str) -> str:
```

Comment:
The cap should be 2048, not 2000. And there should be an indicator at the end showing that the query was truncated, like `…` (U+2026). The return value should be never be longer than 2048 and inputs that are exactly 2048 should not be modified.